Middleware/src/test/java/ca/concordia/ngci/tools4cities/middleware/test/TestRandomNumberStorage.java

```diff
@@ -0,0 +1,71 @@
+package ca.concordia.ngci.tools4cities.middleware.test;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.Assertions;
+
+import ca.concordia.ngci.tools4cities.middleware.producers.RandomNumberProducer;
+import ca.concordia.ngci.tools4cities.middleware.consumers.IntegerConsumer;
+import ca.concordia.ngci.tools4cities.middleware.middleware.IProducer;
+
+public class TestRandomNumberStorage {
```

Comment:
Please add a comment block to describe the purpose of the test, similarly to what you did in previous PRs